src/main/java/frc/robot/RobotContainer.java

```diff
@@ -42,7 +45,10 @@ public RobotContainer() {
   }
 
   // Bind triggers to Commands
-  private void configureBindings() {}
+  private void configureBindings() {
+    new JoystickButton(controller, 1)
+      .whenHeld(new ArmPIDCommand(arm, 0))
```

Comment:
`.whenHeld()` is deprecated. I think we should be using `controller.button(1).onTrue(new ArmPIDCommand(arm, 0)` instead.